src/TwoBucks/CreateReport.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Create Organized Report Feature US.4
+ *
+ * The CreateReport class is used when the user chooses to
+ * view a report that contains their saved information on their profile,
+ * their performance, goals, along with their personal information. It gives
+ * the user a full overview of everything that deals with the program.
+ *
+ * @author Frino Jais
+ * @version 1.0
+ * @since 04-01-2020
+ */
+
+
+import java.text.DecimalFormat;
+import java.util.Scanner;
+import java.io.*;
+import java.util.*;
+import java.lang.*;
+import java.time.format.DateTimeFormatter;
+import java.time.LocalDateTime;
+
+public class CreateReport {
+
+    /**
+     * This method will output a report of the User's full profile and analysis
+     *
+     * @param currentUser
+     */
+    public void showReport(User currentUser){
```

Comment:
Format looks good. For the sake of reusability consider using a String array in order to pass as an object to text file and email file. That way we have less code repetition and the code is reusable.

Reply:
Done, please check my changes.